### DISCUSSION TITLE
Configuration for the Most Access

### DISCUSSION THREAD

**Discussion by Hashnoob** [sandboxie-plus/Sandboxie#1758] [Documents and notes]
When security is not of concern and you wanted to create a sandbox that would give the most direct access to the system other than the registry and file system for keeping things portable, what would the ini settings look like?

**Comment by DavidXanatos** [comment]
Well ideally you want for the 1.0.0 release with its custom box modes like app compartment that has minimal restrictions, 

but till then you can try

BoxNameTitle=-
NoAddProcessToJob=y
NoSandboxieDesktop=y
OriginalToken=y
NoSysCallHooks=y
OpenIpcPath=*
OpenWinClass=*

Note that with this config you have no COM isolation i.e. windows installers (MSI installers) will always run outside the sandbox and install things outside (!!!)
You can remove 'OpenIpcPath=*'  to make MSI installer run inside the box

Note that this configuration does not provide a reliable isolation, processes can use IPC to escape the sandbox.
